content/blog/benefits-of-the-platform-level-data-model-for-firmware-update-standard.md

```diff
@@ -0,0 +1,101 @@
+
+In 2016, the [Distributed Management Task Force](https://dmtf.org) published the first version of the Platform Level Data Model for Firmware Update Specification ([PLDM for FWUPD](https://www.dmtf.org/dsp/DSP0267)). This DMTF standard started to be implemented in the Redfish® service of HPE iLO 5 version 2.30.
+
+The best definition of PLDM for FWUPD is provided in the introduction paragraph of the [DSP0267](https://www.dmtf.org/sites/default/files/standards/documents/DSP0267_1.1.0.pdf) specification document:
+
+"*The Platform Level Data Model (PLDM) Firmware Update Specification defines messages and data structures for updating firmware or other code objects maintained within the firmware devices of a platform management subsystem. Additional functions related to the sequence of identifying and transferring the firmware, are also defined.*"
+
+This article explains how this standard simplifies and enhances the update firmware process of servers implementing it for the benefit of end customers, firmware suppliers and computer makers.
+
+For more architecture detail concerning PLDM, read the [Overview of the Platform Level Data Model for Redfish Device Enablement Standard](https://developer.hpe.com/blog/overview-of-the-platform-level-data-model-for-redfish%C2%AE-device-enablement-standard/) (PLDM for RDE) blog post.
+
+## Classification of the different firmware update types
+
+Computer firmware update is a complex process involving several types of objects in different contexts. An attempt to explain how the Redfish service of HPE iLO implements this process is presented in the following three blog posts:
+
+* [HPE firmware updates: Part 1 - File types and Smart Components](https://developer.hpe.com/blog/hpe-firmware-updates-part-3-the-redfish-update-service/)
+* [HPE firmware updates: Part 2 - Interaction in operating modes](https://developer.hpe.com/blog/hpe-firmware-updates-part-2-interaction-in-operating-modes/)
+* [HPE firmware updates: Part 3 - The Redfish update service](https://developer.hpe.com/blog/hpe-firmware-updates-part-3-the-redfish-update-service/)
+
+Although the above articles provide many definitions, acronyms and their relationship with the firmware update process, I will use, in this blog post, a different approach to this process than the one used in the three articles mentioned above.
+
+Firmware updates can be classified in four types called A, B, C and D. This classification is HPE specific but you may read or hear references to it, like in the [HPE Redfish API Reference Document](https://hewlettpackard.github.io/ilo-rest-api-docs/ilo5/#configuration).
+
+### Type A firmware update
+
+In this type of firmware update, the firmware can be flashed to its destination component by the iLO and become active without any server reboot required.
+
+Examples of firmware falling in this category:
+
+* iLO firmware
+* Power Supply firmware
+* PLDM firmware
+
+> Note: The last bullet of this list is the reason of this article.
```

Comment:
Now you have me really intrigued... but you leave me hanging. Please... tell me more here!

Reply:
Removed the "> Note:", as it generates too much confusion.